nucleus/diagnostics/context/src/test/java/org/glassfish/diagnostics/context/impl/ContextManagerImplUnitTest.java

```diff
@@ -77,7 +77,8 @@ public void registerContextFactoryForPrefixNamed(String prefixName, ContextViewF
 			}
 		};
 
-		ContextManagerImpl cmi = new ContextManagerImpl();
+		@SuppressWarnings("unused")
```

Comment:
Is this correct? It appears to be briefly used on line 116

Reply:
I'm not sure, it's in the original test, and potentially it tests whether the instance can be created without throwing exceptions.